src/main/java/Ui.java

```diff
@@ -0,0 +1,31 @@
+//package duke;
+/**
+ * interact with user
+ */
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    /**
+     * greet user
+     */
+    public void sayHi() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * bye user
+     */
```

Comment:
Perhaps you could replace the JavaDoc method header comment for this method as "Says bye to user"? This was mentioned in the guidelines for Java coding standard which mentioned that "In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)".